src/client/components/forms/parts/achievement.tsx

```diff
@@ -57,8 +57,37 @@ const maxAchievementProgress = {
 	30: 100
 };
 
-function Achievement(props) {
-	const {achievement, counter, unlocked} = props;
+interface Props {
+	achievement: {
+		id: number;
+		name: string;
+		description: string;
+		badgeUrl: string;
+	},
```

Comment:
We have a type already defined for achievement which can be imported.
It is currently being improved in #1033 (the `badgeUrl` prop needs fixing) so you will need to wait until that PR is merged before importing it